en/config/battery.md

```diff
@@ -0,0 +1,81 @@
+
+## With load compensation
+If you use load compensation the empty voltage should be set higher than without because the compensated voltage gets used for the estimation. What you want to set is typically a bit below the expected rest cell voltage when empty after use.
+
+### With thrust based load compensation (not recommended)
+Without a current sensor it's very hard to do load compensation. You can enable compensation based on the total thrust which gets commanded to the motors by setting the parameter BAT_V_LOAD_DROP to how much voltage drop a cell shows under the load of full throttle. But this method is not accurate even with a good parameter because there's a delay between thrust command and current and the thrust in not linearly proportional to the current.
+
+### With current based load compensation (recommended)
+If you have a current sensor on board you can enable load compensation using the current measurement by setting the parameter `BAT_R_INTERNAL` to the internal resistance of your battery. There are li-po chargers out there which can measure the internal resistance of your battery. A typical value is 5mΩ but this can vary with discharge current rating, age and health of the cells.
+With decent configured load compensation the voltage used for the battery fuel estimation is much more stable and will not vary that much anymore when flying up and down.
+
+### Calibration Voltage Devider & Current per volt
+If you have a vehicle which measures voltage and current through a power module and the ADC of the FCU then you should check and calibrate the measurements once per board. To calibrate you'll need a multimeter.
+
+The easiest way to calibrate the dividers is by using QGroundControl and following the step-by-step guide on https://docs.qgroundcontrol.com/en/SetupView/Power.html.
+
+## Combning voltage based estimation with current integration
+After tuning the voltage based estimation you can achieve a more stable linear and accurate estimation by fusing your estimate with the so far used capacity. This is only possible with accurate current measurements because the current gets mathematically integrated over time to have a used capacity estimate.
+This is the best way to have totally accurate relative fuel consumption of the battery. You can not use only this technique because:
+* You cannot detect how full the battery is when the vehicle is turned on
+* If the absolute fuel estimation is for some reason wrong or the battery is faulty you don't have feedback.
+
+That's why you should always configure and tune the voltage based estimation first and then afterwards enable direct used capacity fusion.
+To enable this feature make sure you have a working and safe voltage based estimation then set the parameter `BAT_CAPACITY` to around 90% of the advertised battery capacity which is usually printed on a label on the battery.
```

Comment:
So what I think you are saying is that if you set BAT_CAPACITY and you have a current sensor, PX4 will estimate fuel consumption and let you know approximately how much you have left. You're also saying that you need to always have a full battery when you start so that you can set the capacity correctly. 
- Is that right?
- If you don't set the BAT_CAPACITY what changes? PX4 just won't tell you how much fuel you have left in warnings?